lib/identity_cache/replicated_cache_proxy.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module IdentityCache
+  class ReplicatedCacheProxy
```

Comment:
Should we have a lower expiration on these than the default `6.hours` in IDC? It seems like having separate cache entries out of sync would be 😬 , at least currently you won't flip-flop. Perhaps it's fine, especially if we can already override it per model.